src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -1,6 +1,5 @@
 {# Table Header #}
-{% if table.option('title') or view is defined and view.definition.hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) or table.filterExtension and table.filterExtension.filters|length > 0 or table.searchExtension and table.option('searchable') %}
-
+{% if table.option('title') or view is defined and table.option('definition').hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) or table.filterExtension and table.filterExtension.filters|length > 0 or table.searchExtension and table.option('searchable') %}
```

Comment:
please group the `and` and `or` in brackets for better readability

Reply:
It's now more readable